apitools/base/py/credentials_lib.py

```diff
@@ -302,6 +302,9 @@ def _CheckCacheFileForMatch(self, cache_filename, scopes):
                             if (creds['scopes'] in
                                     (None, cached_creds['scopes'])):
                                 scopes = cached_creds['scopes']
+                except:  # pylint: disable=bare-except
```

Comment:
Is it not possible to list specific exceptions we care to skip. This would also supress Ctrl-C.